app/scripts/lib/stores/extension-store.test.ts

```diff
@@ -0,0 +1,93 @@
+import browser from 'webextension-polyfill';
+import ExtensionStore from './extension-store';
+
+const MOCK_STATE = { data: {}, meta: { version: 1 } };
+
+jest.mock('webextension-polyfill', () => ({
+  runtime: { lastError: null },
+  storage: { local: true },
+}));
+
+const setup = (
+  options: { localMock?: { get?: unknown; set?: unknown } | false } = {},
+) => {
+  if (typeof options.localMock === 'undefined') {
+    // @ts-expect-error Mock used just to spy on calls, doesn't implement API
+    jest.replaceProperty(browser.storage, 'local', jest.fn());
+  } else if (options.localMock === false) {
+    const storageApi: Partial<typeof browser.storage> = { ...browser.storage};
+    delete storageApi.local;
+    // @ts-expect-error Intentionally incomplete to test behavior when API is missing
+    jest.replaceProperty(browser, 'storage', storageApi);
+  } else {
+    // @ts-expect-error Incomplete mock, it just has the properties we call
+    jest.replaceProperty(browser.storage, 'local', options.localMock);
+  }
+  return new ExtensionStore();
+};
+describe('ExtensionStore', () => {
+  beforeEach(() => {
+    jest.replaceProperty(global.sentry, 'captureException', jest.fn());
```

Comment:
Unit tests seem to be failing due to this line. This expects `global.sentry` to be defined, but it's not. We could update it to something like this:

```suggestion
    jest.replaceProperty(global, 'sentry', { captureException: jest.fn() });
```

(didn't try this, this might introduce type errors, not sure)